rfc_as_pr.md

```diff
@@ -0,0 +1,22 @@
+# RFC 9
+
+## Summary
+
+Use PRs rather than issues for the RFC process.
+
+## Details
+
+Each RFC should be a markdown-formatted document in the rfcs/ subdirectory. Creating the PR will signal the the RFC is ready for review and merging it will indicate that it's accepted. Conversely closing it will be used to indicate the RFC is rejected.
+The RFC document should get a title that is its PR number (added after submission) and consists of a summary of the proposed changes, full details of the proposal and a discussion of pros and cons.
```

Comment:
I note that the PR number isn't part of the filename, and that seems fine to me!